src/rldp_util.erl

```diff
@@ -14,5 +14,5 @@ cmd_in_dir(Cmd, Dir) ->
     end.
 
 mktemp_name(Prefix) ->
-    <<Int:32/big-unsigned-integer>> = crypto:rand_bytes(4),
+    <<Int:32/big-unsigned-integer>> = crypto:strong_rand_bytes(4),
```

Comment:
When was this function introduced? Assuming that switching it blindly might create an incompatibility with an older Erlang version.